addons/website/views/website_templates.xml

```diff
@@ -197,7 +197,7 @@
                 <input t-if="optionName in main_object" type="hidden" class="o_page_option_data" autocomplete="off" t-att-name="optionName" t-att-value="main_object[optionName]"/>
             </t>
         </t>
-        <div groups="base.group_user" class="o_frontend_to_backend_nav position-fixed d-none">
+        <div groups="base.group_user" class="o_frontend_to_backend_nav position-fixed d-none d-print-none">
```

Comment:
I am not a fan of modifying directly based website templates ... could have unwanted side effects. Can't you make a rule in media print of survey template scss file instead ? That way we keep changes localized, and we don't have to think of all possible use cases involving website ^^